open-machine-learning-jupyter-book/ml-advanced/model-selection.ipynb

```diff
@@ -49,7 +49,9 @@
     "\n",
```

Comment:
<p>Same, like Accuracy，Precision and Recall and so on , they can be displayed in bold to ensure readability.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>